server/tserver/src/main/java/org/apache/accumulo/tserver/scan/LookupTask.java

```diff
@@ -177,9 +177,13 @@ public void run() {
       }
     } catch (SampleNotPresentException e) {
       addResult(e);
-    } catch (Throwable e) {
+    } catch (Exception e) {
       log.warn("exception while doing multi-scan ", e);
       addResult(e);
+    } catch (Error t) {
+      log.warn("Error while doing multi-scan ", t);
+      addResult(t);
```

Comment:
Is there value in adding this to the result if we're re-throwing?

Reply:
I was following what the other catch clauses were doing here. I assume that the addResult() call is putting the exception on the result queue to be sent back to the client. It's possible that this error could make it's way back to the client before the tserver is halted.

Reply:
If there's an actual Error, we probably shouldn't leak those details to the client. The client will get a TApplicationException when the server fails. It's probably not worth creating new work before the Error falls through and is ultimately handled. We probably also don't need the log message, either, because the code that will eventually handle the Error should provide a useful message/stack trace, and this would be redundant. It's probably better to just let it fall through and not even try to catch-handle-rethrow.

Reply:
I can remove this Error handling, I was just trying to follow the pattern.